examples/active.ini

```diff
@@ -0,0 +1,39 @@
+[acq_params]
+adc_offset = 33
+amplitude = 1.0
+p90_us = 4.8
+deadtime_us = 10.0
+tau_us = 3500.0
+deblank_us = 1.0
+sw_khz = 3.9
+acq_time_ms = 1024.0
+nscans = 1
+thermal_nscans = 3
+carrierfreq_mhz = 14.895116358000001
+nechoes = 1
+startconstant = 0.15
+stopconstant = 2.0
+gamma_eff_mhz_g = 0.0042419157275600345
+field_width = 10.0
+
+[sample_params]
+concentration = 0.005
+krho_cold = 380.0
+krho_hot = 260.0
+t1water_hot = 2.98
+t1water_cold = 2.17
```

Comment:
if an example for hydroxytempo: say so at the top of the file, and state what needs to be changed (normal python comment -- will be removed on re-saving file)